TestFoundation/Utilities.swift

```diff
@@ -203,7 +224,11 @@ extension Fixture where ValueType: NSObject & NSCoding {
         let original = try make()
         
         let coder = NSKeyedArchiver(forWritingWith: NSMutableData())
+#if !DARWIN_COMPATIBILITY_TESTS
+        // FIXME: On Darwin, this is defined as @property (readwrite) NSDecodingFailurePolicy decodingFailurePolicy
```

Comment:
Something is very wrong here. The current definition of this property on master is:

```
open var decodingFailurePolicy: NSCoder.DecodingFailurePolicy = .raiseException
```

Reply:
This is when running against native Foundation. On Xcode 10.2.1 the definition comes from :

```
/Applications/Xcode.app//Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX.sdk/System/Library/Frameworks/Foundation.framework/Versions/C/Headers/NSKeyedArchiver.h

@property (readwrite) NSDecodingFailurePolicy decodingFailurePolicy API_AVAILABLE(macos(10.11), ios(9.0), watchos(2.0), tvos(9.0));
```

Reply:
I think I've fixed this correctly in https://github.com/apple/swift-corelibs-foundation/pull/2324